clients/ui/frontend/.env.cypress.mock

```diff
@@ -0,0 +1,2 @@
+# Test against prod build hosted by lightweight http server
+BASE_URL=http://localhost:9001
```

Comment:
We'll need to do a follow up to better handled .env variables.

Reply:
I've created a follow up doc to handle this, but it's more than enough right now.